.bazelrc_shared

```diff
@@ -8,20 +8,13 @@ build --tool_java_runtime_version="remotejdk_21"
 # Other options
 build --experimental_use_hermetic_linux_sandbox
 build --experimental_worker_sandbox_hardening
+build --experimental_worker_multiplex_sandboxing
```

Comment:
Super-nit but would you mind re-sorting these? I'm trying to keep our Bazel options organized.

Reply:
Yep. Will do.